.env

```diff
@@ -1,16 +1,30 @@
+## See the "Settings" section in README.md for more details
+
 ## Set this to true if you're using a load balancer, or set it to false if you're using seperate IPs for each service.
+## If you're using monolithic (the default), leave this set to true
 USE_GENERIC_CACHE=true
 
-## IP Address that the lancache load balancer will be running on
+## IP Address that the lancache monolithic instance will run on
+## Specify multiple IPs, space separated, if your lancache host has them - this can improve cache priming performance for some services (e.g. Steam)
```

Comment:
@Lepidopterist pointed out there was some confusion if you just read the .env as this isn't the IP the cache will "bind" to, but the ip's the dns will serve. The wording makes it seem 0.0.0.0 is possible to bind to "everything"

Reply:
Good point. How about this for wording:
```yaml
## IP addresses that the lancache monolithic instance is reachable on
## Specify one or more IPs, space separated - these will be used when resolving DNS hostnames through lancachenet-dns. Multiple IPs can improve cache priming performance for some services (e.g. Steam)
```

Reply:
Looks good, but add the following 3rd line
`
Note: This setting only effects dns, monolithic will still bind to all IP's by default
`

Reply:
Changed.